src/convertToObject.js

```diff
@@ -5,8 +5,30 @@
  *
  * @return {object}
  */
+
 function convertToObject(sourceString) {
-  // write your code here
+  const delEmptyLine = sourceString.replace(/\n/g, '');
+
+  const commandsArr = delEmptyLine.split(';');
+
+  const commandsArrFinal = commandsArr.filter((str) => str.trim());
+
+  const resultObject = {};
+
+  for (let i = 0; i < commandsArrFinal.length; i++) {
+    const commandArr = commandsArrFinal[i].split(':');
+
+    const key = commandArr[0].trim();
+    const value = commandArr[1].trim();
+
+    resultObject[key] = value.replace(/,/g, ',\n');
```

Comment:
[CODE KNOWLEDGE] - Don't use iteration methods to modify an already existing array/object. (forEach is exception). These methods return a new array, so you should use it. Instead of manually building `resultObject`, you can use `reduce` to transform `commandsArrFinal` into the desired object.